### PR TITLE
resolving pr comments

### PR DESCRIPTION
**Pull request by mehtadarshan1**

### CODE REVIEW

**Review by esteeroramy — APPROVED** [review]

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Looks good

File: server/questions.js [hunk 1]
```diff
@@ -146,19 +146,15 @@ var validateQuestionByType = function(question, type){
 }
 
 var multipleChoiceValidator = function(question){
-	if (question.choices){
-		if (question.choices.length < 2){
-			return 'Need two or more options for Multiple Choice Question';
-		}
+	if (question.choices && question.choices.length < 2){
+		return 'Need two or more options for Multiple Choice Question';
```
Comment (ahmedqarmout2) [inline server/questions.js:150]: If no choices supplied, the verification would pass. We should check !question.choices || question.choices.length<2

Comment (ahmedqarmout2) [reply server/questions.js:150]: Also make this return true or false

File: server.js [hunk 2]
```diff
@@ -304,7 +304,7 @@ app.get('/answerForm', function(req, res){
                 answerForm:true});
             break;
         default:
-            return res.redirect('/');
+            return res.status(400).send('Please select an appropriate question Type.')
```
Comment (ahmedqarmout2) [inline server.js:307]: Nice. I think there is another place for this. I will have to check my previous comments

**Review by ahmedqarmout2 — CHANGES_REQUESTED** [review]

File: server/questions.js (hunk 1 above)
Comment (mehtadarshan1) [reply server/questions.js:150]: Can't return true/false, the function is returning specific status back to the user. (err message)

**Review by Mohammed-JS-Khan — APPROVED** [review]
Good job :+1:

File: server/questions.js (hunk 1 above)
Comment (ahmedqarmout2) [reply server/questions.js:150]: You are right, my bad. Overall it looks good, I just need to double check my previous comments to make sure we are not missing anything

**Review by ahmedqarmout2 — APPROVED** [review]

**Pull request merged**